src/assets/thirdPartyServices/index.js

```diff
@@ -103,7 +103,7 @@ const addRejuveCustomUI = (serviceId, CustomUIComponent) => {
 };
 
 const addNunetCustomUI = (serviceId, CustomUIComponent) => {
-  thirdPartyCustomUIComponents.addCustomUIComponent("nunet", serviceId, CustomUIComponent);
+  thirdPartyCustomUIComponents.addCustomUIComponent("nunet-org-new", serviceId, CustomUIComponent);
```

Comment:
You have to add the mapping for both `nunet` and the `nunet-org-new` here as we have different org name for dev and master in case of nunet.